src/pages/Mavanimes/main.ts

```diff
@@ -0,0 +1,59 @@
+import { pageInterface } from '../pageInterface';
+
+export const Mavanimes: pageInterface = {
+  name: 'Mavanimes',
+  domain: 'http://www.mavanimes.co/',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (j.$('select').length) {
+      return true;
+    }
+    return false;
+  },
+  
+
+  sync: {
+    getTitle(url) {
+      var title = j.$('meta[property="article:section"]').attr('content');
```

Comment:
Does not allways contain the title:
http://www.mavanimes.co/blue-exorcist-17-vf/